src/commands/configure-docker-credentials-store.yml

```diff
@@ -0,0 +1,12 @@
+description: >
+  Install the ACR docker credential helper and configure Docker to use
+  it for its credentials store.
+  Requires bash and Docker to be available.
+
+steps:
+  - run:
+      name: Configure Azure ACR helper
+      command: |
+          # Run ACR Docker Credential Helper installation script
+          # https://github.com/Azure/acr-docker-credential-helper#installation
+          curl -L https://aka.ms/acr/installaad/bash | /bin/bash
```

Comment:
~will this ever need sudo?~

Nm! The install script has sudo check ,logic baked in!